### ISSUE TITLE
Pretrained Resnet18 trainable false doesnot freeze base layers

### ISSUE THREAD

**Issue by mariyamuneeb** [qubvel/classification_models#41]
resnet18, preprocess_input = Classifiers.get('resnet18')
base_model = resnet18(input_shape=(224,224,3), weights='imagenet', include_top=False,trainable=False)
x = keras.layers.GlobalAveragePooling2D()(base_model.output)
output = keras.layers.Dense(10, activation='softmax')(x)
model = keras.models.Model(inputs=base_model.input, outputs=output)
model.compile(optimizer='SGD', loss='categorical_crossentropy', metrics=['accuracy'])
model.summary()

The output of the above snippet taken from pretrained finetuning in ReadMe give the following summary of trainable and non-trainable params. The numbers seem off even if trainable is set to False
Total params: 11,192,019
Trainable params: 11,184,077
Non-trainable params: 7,942

**Comment by FerdinandZhong** [comment]
I encountered the same issue. May i know if you have found a solution?

**Comment by ananthsadanand** [comment]
Same issue. Has anybody been able to solve this?
